validator/tests/test_merkle_trie/tests.py

```diff
@@ -78,31 +78,47 @@ def test_merkle_trie_delete(self):
     def test_merkle_trie_update(self):
         init_root = self.get_merkle_root()
 
+        values = {}
         key_hashes = {
             key: _hash(key) for key in
             (_random_string(10) for _ in range(1000))
         }
 
         for key, hashed in key_hashes.items():
+            value = {key: _random_string(512)}
             new_root = self.set(
-                hashed,
-                {key: _random_string(512)})
+                hashed, value, ishash=True)
```

Comment:
Not sure where this parameter name is coming from originally, but it should be `is_hash`.